VHostScan/lib/helpers/color_helper.py

```diff
@@ -0,0 +1,104 @@
+from termcolor import colored
+
+__COLOR_ENABLED = True
+
+text_settings = {
+    'process':      ('cyan', []),
+    'result_head':  ('magenta', []),
+    'result':       ('white', []),
+    'key':          ('yellow', []),
+    'link':         ('green', ['underline']),
+    'error':        ('red', ['bold']),
+    'warning':      ('red', ['bold', 'dark']),
+}
+
+all_colors = [
+    'grey',
+    'red',
+    'green',
+    'yellow',
+    'blue',
+    'magenta',
+    'cyan',
+    'white'
+]
+
+"""
+avalable attributes
+    'bold'
+    'dark'
+    'underline'
+    'blink'
+    'reverse'
+    'concealed'
+"""
+
+
+def config_colorization(no_color):
+    global __COLOR_ENABLED
+    __COLOR_ENABLED = not no_color
+
+
+def __convert_to_color(value, color, attrs):
+    if __COLOR_ENABLED:
+        return colored(str(value), color, attrs=attrs)
+    else:
+        return str(value)
+
+# color-based converter functions
+for color in all_colors:
+    exec('def t_{} (value, attrs=[]): '.format(color) +
+         'return __convert_to_color(value, "{}", attrs)'.format(color))
+
+for purpose, args in text_settings.items():
+    exec('def t_{} (value): '.format(color) +
+         'return t_{}(value, attrs={})'.format(*args))
```

Comment:
I know this is really novel, but using exec is almost always a bad idea. Can we achieve something similar using classes?

Reply:
Got it! Turning that part into a class will make it more readable too. I'll do shortly.